.buildkite/pipeline.agentless-app-release.yaml

```diff
@@ -1,13 +1,56 @@
 env:
   VERSION: "${BUILDKITE_COMMIT:0:12}"
+
+# This section is used to define the plugins that will be used in the pipeline.
+# See https://buildkite.com/docs/pipelines/integrations/plugins/using#using-yaml-anchors-with-plugins
+common:
+  - docker_login_plugin: &docker_login_plugin
+      elastic/vault-docker-login#v0.5.2:
+        secret_path: 'kv/ci-shared/platform-ingest/elastic_docker_registry'
+
 steps:
-  - label: "Mirror Elastic-Agent Snapshot DRA to internal registry"
-    key: "mirror-elastic-agent"
-    command: ".buildkite/scripts/steps/ecp-internal-release.sh"
+  - label: "Packaging: Service Container linux/amd64"
+    key: packaging-service-container-amd64
+    env:
+      PACKAGES: "docker"
+      PLATFORMS: "linux/amd64"
+      DOCKER_VARIANTS: "service"
+    command: |
+      .buildkite/scripts/steps/integration-package.sh
```

Comment:
I'm not sure if the name of the script is correct here, IIUC, that script is used by the integrations pipeline, hence the reason for that name.

It was introduced at https://github.com/elastic/elastic-agent/pull/5129/files

I wonder if the name should be different. I'm not very familiar with the specifics of agentless, but it might cause some issues if it's changed for other reasons and there are multiple consumers.

Reply:
@v1v I use that script because we want the image built the same way that the integration steps verified the image. I can duplicate it and name it differently, but re-using that script was kinda a point of saying we CI validate the same build, so we will use the same build.

I can add a comment in the file to ensure that its clear that its used in both, would that work? Or do you think duplicating the file with a different name is better?